### ISSUE TITLE
isinstance(module, torch.nn.BatchNorm2d)

### ISSUE THREAD

**Issue by jackowang** [eeric/channel_prune#6]
Is their any pros and cons between doing process on conv vs batchnorm? How do you comment ?

if isinstance(module, torch.nn.modules.conv.Conv2d): #or isinstance(module, torch.nn.BatchNorm2d)

**Comment by eeric** [comment]
Conv was usually pruned, while batchnorm was modified according to output channels of Conv.
so,it was used that isinstance(module, torch.nn.modules.conv.Conv2d).
